src/app/containers/MostRead/Canonical/lastUpdated.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Rename file name to be LastUpdated

Reply:
Tried to do this just now, but experiencing the horrors of MacOS being case-insensitive.